plugins/reveal.js

```diff
@@ -46,10 +46,11 @@ class Reveal {
   configure() {
     return this.page.evaluate(config => {
         Reveal.configure({
-          controls  : false,
-          progress  : config.progress,
-          fragments : config.fragments,
-          transition: 'none'
+          controls   : false,
+          progress   : config.progress,
+          fragments  : config.fragments,
+          transition : 'none',
+          autoAnimate: false
```

Comment:
nit: could you add a trailing comma?

Reply:
Done. I removed it because the previous code didn't have a trailing comma 😅